src/util/lib/cosi/registration/dashboards/index.js

```diff
@@ -223,7 +223,7 @@ class Dashboards extends Registration {
         console.log(chalk.blue(this.marker));
         console.log(`Configuring dasbhoard`);
 
-        const templateMatch = template.file.match(/^template-dashboard-([^-]+)-(.+)\.json$/);
```

Comment:
why is the dash being removed? the format for template files is template-<type>-<id>